eng/pipeline-pr.yml

```diff
@@ -167,13 +167,14 @@ jobs:
 
       # Use utility script to run script command dependent on agent OS.
       - script: eng\scripts\cibuild.cmd
-          -configuration $(_BuildConfig) 
+          -configuration $(_BuildConfig)
           -prepareMachine
           $(_PublishArgs)
           $(_SignArgs)
           $(_OfficialBuildIdArgs)
           $(_PlatformArgs)
           $(_InternalRuntimeDownloadArgs)
+          /bl:$(Build.SourcesDirectory)\artifacts\log\$(_BuildConfig)\BuildSrc-$(_Platform).binlog
```

Comment:
Why is this needed? The Arcade default that most/all(?) repos follow is to just produce a Build.binlog file.

Reply:
Yes, please undo